pkg/server/server.go

```diff
@@ -401,6 +414,25 @@ func (s *Server) VerifyEmail(c *gin.Context, redirectURL string) {
 	jwtUserCredentials := jwtUser.(*models.UserCredentials)
 
 	if jwtUserCredentials.UnverifiedEmail != "" {
+		// Checking if the given email is already registered with some user
```

Comment:
Code seems to be repeated above.

Reply:
Only the check is repeated. Inside the if blocks we have redirect statements here while in the above code no redirection is happening.